### PR TITLE
Duplicate research output

### PR DESCRIPTION
**Pull request by kmaid**

### CODE REVIEW

File: packages/react-components/src/templates/SharedResearchOutput.tsx [hunk 1]
```diff
@@ -108,11 +130,11 @@ const SharedResearchOutput: React.FC<SharedResearchOutputProps> = ({
         </div>
       )}
       <div css={containerStyles}>
-        <div css={buttonsContainer}>
-          <BackLink href={backHref} />
```
Comment (gabiayako) [inline packages/react-components/src/templates/SharedResearchOutput.tsx:112]: Removing the BackLink is intentional?

Screenshot 2023-05-17 at 07 26 24

File: apps/crn-frontend/src/network/teams/__tests__/TeamProfile.test.tsx [hunk 2]
```diff
@@ -150,38 +161,193 @@ it('does not allow navigating to the workspace tab when team tools are not avail
...
-    { teamId: teamResponse.id, currentTime: new Date() },
-    {
-      ...userResponse,
-      teams: [
-        {
-          ...userResponse.teams[0],
-          id: teamResponse.id,
-          role: 'ASAP Staff',
-        },
+  it('does not show the share outputs button when the user does not have', async () => {
```
Comment (gabiayako) [inline apps/crn-frontend/src/network/teams/__tests__/TeamProfile.test.tsx:200]:
```suggestion
  it('does not show the share outputs button when the user is not a member of the team', async () => {
```

File: apps/crn-frontend/src/network/teams/__tests__/TeamProfile.test.tsx [hunk 3]
```diff
@@ -150,38 +161,193 @@ it('does not allow navigating to the workspace tab when team tools are not avail
     screen.queryByText(/workspace/i, { selector: 'nav *' }),
   ).not.toBeInTheDocument();
 });
+describe('Share Output', () => {
+  it('shows share outputs button and page when the user has permissions user clicks an option', async () => {
```
Comment (gabiayako) [inline apps/crn-frontend/src/network/teams/__tests__/TeamProfile.test.tsx:165]:
```suggestion
  it('shows share outputs button when the user has permissions', async () => {
```

**Review by gabiayako — APPROVED** [review]

File: packages/react-components/src/templates/SharedResearchOutput.tsx (hunk 1 above)
Comment (kmaid) [reply packages/react-components/src/templates/SharedResearchOutput.tsx:112]: Yes, thats where the duplicate buttons go. I had this confirmed Jao as they were omitted from the designs

**Pull request merged**